plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk

```diff
@@ -952,6 +953,15 @@ advanced option portSortingStrategy: PortSortingStrategy {
     targets parents
 }
 
+advanced option preserveOrder: OrderingStrategy {
+    label "Preserve Order"
```

Comment:
Can we find a more speaking name for this option? As it is one may ask "which order"?

Maybe 
* Consider model order
* Take model element order into account

Reply:
`considerModelOrder` seems to fit.